models/cow_protocol/ethereum/cow_protocol_ethereum_sources.yml

```
@@ -0,0 +1,38 @@
+version: 2
+
+sources: 
+  - name: gnosis_protocol_v2_ethereum
+    description: "Ethereum decoded tables related to CoW Protocol contract"
+    tables:
+      - name: GPv2AllowListAuthentication_evt_SolverAdded
+        description: "GPv2AllowListAuthentication list of events when a Solver was added"
+        columns:
+          - &solver
+            name: solver
+            description: "Solver wallet address"
+          - &contract_address
+            name: contract_address
+            description: "Ethereum address for the liquidity pool used in transaction"
```

I guess this was copied from elsewhere.
```suggestion
            description: "Ethereum address of contract emitting the event"
```